consensus/misc/eip1559/eip1559.go

```diff
@@ -132,6 +132,53 @@ func ValidateHoloceneExtraData(extra []byte) error {
 	return ValidateHolocene1559Params(extra[1:])
 }
 
+// DecodeMinBaseFeeExtraData decodes the extraData parameters from the encoded form defined here:
+// https://github.com/ethereum-optimism/design-docs/blob/main/protocol/minimum-base-fee.md#minimum-base-fee-in-block-header
```

Comment:
Ideally this would be a specs link, let's try and get the linked material copied over to the specs repo.